### PR TITLE
Admin lists

### PR DESCRIPTION
**Pull request by Macroz**
Implements #805

### CODE REVIEW

File: src/cljs/rems/administration/forms.cljs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+   (or (::sorting db)
+       {:sort-column :title
+        :sort-order :asc})))
+
+(defn- to-create-form []
+  [:a.btn.btn-primary
+   {:href "/#/create-form"}
+   (text :t.administration/create-form)])
+
+(defn- forms-columns [language]
```
[the remark this answers is not in the document]
Comment (Macroz) [reply src/cljs/rems/administration/forms.cljs]: I removed them now.

File: src/cljs/rems/spa.cljs [hunk 2]
```diff
@@ -271,28 +281,48 @@
...
+  (rf/dispatch [:rems.administration.create-license/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-license]))
+
+(secretary/defroute "/administration/create-resource" []
+  (rf/dispatch [:rems.administration.create-resource/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-resource]))
+
+(secretary/defroute "/administration/create-workflow" []
+  (rf/dispatch [:rems.administration.create-workflow/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-workflow]))
```
Comment (luontola) [inline src/cljs/rems/spa.cljs]: `rems.navbar/navbar-items` needs to be updated for the administration link to be highlighted correctly. Rename the old page IDs and add the new ones.

File: src/cljs/rems/administration/catalogue_items.cljs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+   :form {:header #(text :t.administration/form)
+          :value :form-name}
+   :workflow {:header #(text :t.administration/workflow)
+              :value :workflow-name}
+   :created {:header #(text :t.administration/created)
+             :value (comp localize-time :start)}
+   :end {:header #(text :t.administration/end)
+         :value (comp localize-time :end)}
+   :active {:header #(text :t.administration/active)
+            :value (comp readonly-checkbox :active)}
```
Comment (luontola) [inline src/cljs/rems/administration/catalogue_items.cljs]: All catalogue items are shown as inactive. They don't have an `:active` property.

File: src/cljs/rems/administration/licenses.cljs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  {:title {:header #(text :t.administration/licenses)
+           :value :title}
+   :type {:header #(text :t.administration/type)
+          :value :licensetype}
+   :start {:header #(text :t.administration/created)
+           :value (comp localize-time :start)}
+   :end {:header #(text :t.administration/end)
+         :value (comp localize-time :end)}
+   :active {:header #(text :t.administration/active)
+            :value (comp readonly-checkbox :active)}
```
Comment (luontola) [inline src/cljs/rems/administration/licenses.cljs]: All licenses are shown as inactive. They don't have an `:active` property.

File: src/cljs/rems/spa.cljs (hunk 2 above)
Comment (Macroz) [reply src/cljs/rems/spa.cljs]: :+1:

File: src/cljs/rems/administration/catalogue_items.cljs (hunk 3 above)
Comment (Macroz) [reply src/cljs/rems/administration/catalogue_items.cljs]: :ballot_box_with_check:

File: src/cljs/rems/administration/licenses.cljs (hunk 4 above)
Comment (Macroz) [reply src/cljs/rems/administration/licenses.cljs]: Yes not yet but they need to be implemented [here[(https://github.com/CSCfi/rems/issues/806) so I'll leave the column.

File: src/cljs/rems/administration/administration.cljs [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+   (text :t.administration/workflows)])
+
+(defn- to-licenses [class]
+  [:a {:class class
+       :href "/#/administration/licenses"}
+   (text :t.administration/licenses)])
+
+(defn administration-navigator [selected]
+  [:div.navbar.mb-4
+   [to-administration [:nav-item :nav-link (when (= selected :administration) :active)]]
```
Comment (luontola) [inline src/cljs/rems/administration/administration.cljs]: `:administration` needs to be `:rems.administration/administration`

File: src/cljs/rems/administration/administration.cljs [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+  Subscribes to current page to show the link as selected. The pure functional version is `administration-navigator`"
+  []
+  (let [page (rf/subscribe [:page])]
+    [administration-navigator @page]))
+
+(defn administration-page []
+  (let [loading? (rf/subscribe [::loading?])]
+    (fn []
+      [:div
+       [administration-navigator]
```
Comment (luontola) [inline src/cljs/rems/administration/administration.cljs]: This should be `administration-navigator-container`.

**Pull request merged**